docs/blog/newsletter-october-2020.rst

```diff
@@ -0,0 +1,125 @@
+* `Rex Egg <http://www.rexegg.com/>`_: This lighthearted tutorial assumes a beginner's starting point
+* `Regular expression syntax cheatsheet <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Cheatsheet>`_: A list of JavaScript-specific regex syntax by MDN Web Docs
+* `Ryans Tutorials Grep and Regular Expressions <https://ryanstutorials.net/linuxtutorial/grep.php>`_: When you're ready to pair regex with grep to search text for your regex, try this guide and the accompanying `cheatsheet <https://ryanstutorials.net/linuxtutorial/cheatsheetgrep.php>`_
+* `Advanced Bash Scripting Guide Chapter 18 Regular Expressions <https://tldp.org/LDP/abs/html/regexp.html>`_: If you're comfortable working from the command line, this lesson takes you through a regex overview and provides example commands for grepping a text file
+
+-------------------------------------
+Organizing docs by role, or by topic?
+-------------------------------------
+
+This fall, we fell into a discussion of role-based docs versus topic-based docs, and the reasons to use one over the other. Is it better to organize your documentation by user role or by topic?
+
+With role-based docs, users may spend too much time identifying which type of user you consider them to be. And once they do identify their type, the docs can end up resembling a training course more than a documentation set. A single user can wear many hats, and they then end up needing to jump around more in the docs to do their work; there’s more in `this article from the Nielsen Norman Group <https://www.nngroup.com/articles/audience-based-navigation/>`__. Topic-based docs can allow users to easily go to the topic with the information they need - as long as they can actually find the right topic in the group of topics. 
+
+A sensible way forward can be to combine both ideas. For example, providing a list of links to topics associated with a role; or allowing a user to filter a list of topics by role. It also depends a lot on your specific product, and the audience for it: role-based docs may work well for some products, while topic-based docs may be just the thing for others.
+
+-----------------------
+Command prompts in docs
+-----------------------
+
+Command prompts in docs can be useful visual cues for developers - they provide the context in which to run each command. They also show at a glance that the snippet displays a terminal command and not something else, like a config file. But to avoid mistakes during copying and pasting, our community has these suggestions on integrating the prompt character in documentation:
+
+* Do include prompts if the user modes change or are otherwise relevant to the instructions.
+* Do include prompts for multiple commands in the same snippet, but you might want to avoid prompts for single-line snippets.
```

Comment:
```suggestion
* Include prompts if the user modes change or are otherwise relevant to the instructions.
* Include prompts for multiple commands in the same snippet, but consider avoiding prompts for single-line snippets.
```